contrib/google.golang.org/grpc.v12/grpc.go

```diff
@@ -91,6 +94,7 @@ func UnaryClientInterceptor(opts ...InterceptorOption) grpc.UnaryClientIntercept
 			span ddtrace.Span
 			p    peer.Peer
 		)
+		methodElements := strings.Split(strings.TrimPrefix(method, "/"), "/")
```

Comment:
Nit:
We can use `strings.SplitN(..., 2)` here to avoid more processing and allocations, since we only care about the first part.

Same above.

Reply:
Fixed